hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -51,6 +52,7 @@
     { name: "key1_out", desc: "Passthrough from key1_in, can be configured to invert" }
     { name: "key2_out", desc: "Passthrough from key2_in, can be configured to invert" }
     { name: "pwrb_out", desc: "Passthrough from pwrb_in, can be configured to invert" }
+    { name: "z3_wakeup", desc: "To enter Z3 mode and exit from Z4 sleep mode" }
```

Comment:
Ok, now that these are specified here, we also need to correctly connect them at the top-level.
You can do that in the pinmux config here:
https://github.com/lowRISC/opentitan/blob/4a7399f552e5c3c410db3060ee07e106d9e48fed/hw/top_earlgrey/data/top_earlgrey.hjson#L1233
by adding the two lines:
```
{ instance: "sysrst_ctrl_aon", port: 'lid_open',      connection: 'muxed' , pad: ''             , desc: ''},
{ instance: "sysrst_ctrl_aon", port: 'z3_wakeup',      connection: 'muxed' , pad: ''             , desc: ''},
```

After adding these two lines, you need to rerun topgen by calling `make -C hw top` in the repo root.

@tjaychen OK if we make these muxed for now, or do you think either of them needs to be dedicated?

Reply:
Sure. Will do.

-Eric

Reply:
Pushed the update with the top-level changes. CI is running now.

-Eric